packages/instrumenter/src/mutators/update-operator-mutator.ts

```diff
@@ -0,0 +1,26 @@
+import * as types from '@babel/types';
+import { NodePath } from '@babel/core';
+
+import { NodeMutation } from '../mutant';
+
+import { NodeMutator } from '.';
+
+export class UpdateOperatorMutator implements NodeMutator {
+  public name = 'UpdateOperator';
+
+  private readonly operators: { [targetedOperator: string]: string } = {
```

Comment:
if you make this of type `{ [targetedOperator: string]: '--' | '++; }`, you won't need the `any` cast later.

Reply:
Fixed!